googletest/test/gtest_unittest.cc

```diff
@@ -4188,6 +4188,12 @@ TEST(AssertionSyntaxTest, WorksWithConst) {
 
 }  // namespace
 
+// we don't use the variable further, just avoid compiler warning
+// by defining a function which uses it
+void dummy_use_of_record_property_env() {
```

Comment:
Instead of introducing this dummy function, we could mark the variable with attribute 'unused' in line 2099 (at least for GCC which is complaining here) - more elegant?